docs/installation/prepare/microcontrollers/_index.md

```diff
@@ -26,361 +26,245 @@ Recommended configuration: 384kB Ram + 8MB SPIRAM + 4MB Flash
+Click the **Copy viam-server config** button on the right side of the **Setup** tab of your robot.
+The Micro-RDK uses the config for communication with the Viam app.
+Paste this into your terminal.
 
-## Tips
+{{< alert title="Caution" color="caution" >}}
 
-When developing, you can use the QEMU emulator instead of flashing the ESP32 board.
-Follow these instructions to install and build the emulator.
+All of the generated files should be safe to commit as a project on Github, with the exception of `viam.json`, since it contains a secret key.
 
-### Install Espressif's QEMU ESP32 Emulator
+{{% /alert %}}
 
-Espressif maintains a good [QEMU emulator](https://github.com/espressif/qemu) supporting the ESP32.
+### Upload and Connect to your ESP32 Board
 
 {{< tabs >}}
-{{% tab name="macOS" %}}
-
-Run the following command to install the QEMU ESP32 Emulator:
+{{% tab name="Using Canon" %}}
 
 ```sh {id="terminal-prompt" class="command-line" data-prompt="$"}
-git clone https://github.com/espressif/qemu
-cd qemu
-./configure --target-list=xtensa-softmmu \
-    --enable-gcrypt \
-    --enable-debug --enable-sanitizers \
-    --disable-strip --disable-user \
-    --disable-capstone --disable-vnc \
-    --disable-sdl --disable-gtk --extra-cflags="-I/opt/homebrew/Cellar/libgcrypt/1.10.1/include -I/opt/homebrew//include/"
-cd build && ninja
+canon bash -lc "make build-esp32-bin"
+make flash-esp32-bin
 ```
 
 {{% /tab %}}
-{{% tab name="Linux" %}}
-
-On Ubuntu or Debian, first make sure you have the `libgcrypt` library and headers installed by running the following command:
-
-```sh {id="terminal-prompt" class="command-line" data-prompt="$"}
-sudo apt-get install libgcrypt20 libgcrypt20-dev
-```
+{{% tab name="Local environment" %}}
 
-Then, run the following command to install QEMU:
+Make sure you have sourced `. ~/dev/esp/export-rs.sh` and `. ~/dev/esp/esp-idf/export.sh` prior running the following command
 
 ```sh {id="terminal-prompt" class="command-line" data-prompt="$"}
-git clone https://github.com/espressif/qemu
-cd qemu
-./configure --target-list=xtensa-softmmu     --enable-gcrypt \
-    --enable-debug --enable-sanitizers  --disable-strip --disable-user \
-    --disable-capstone --disable-vnc --disable-sdl --disable-gtk
-cd build && ninja
+make upload
 ```
 
 {{% /tab %}}
 {{% /tabs %}}
 
-Add `export QEMU_ESP32_XTENSA=<path-to-clone-qemu>/build/` to your `.zshrc` or `.bashrc`, or save this command to run in your terminal every session you wish to use the QEMU emulator.
-
-### Build with the QEMU ESP32 Emulator
-
-Navigate to the root of the Micro-RDK repository.
-Once you've `cd`'d to the correct repository, run `. $HOME/esp/esp-idf/export.sh` if you haven't done so already in this terminal session.
-
-You will need to comment out two lines from the file `sdkconfig.defaults`:
+While running above commands, you may be presented with an interactive menu of different serial port options to use to connect to the ESP32 board.

```

Comment:
```suggestion
While running the above commands, you may be presented with an interactive menu of different serial port options to use to connect to the ESP32 board.
```
Why is it may? If it is `will` we can change to that, if not might wanna explain

Reply:
I removed it I don't think it is too critical